aws-ts-eks-hello-world/index.ts

```diff
@@ -60,3 +60,25 @@ export const deployment = new k8s.apps.v1.Deployment(name,
 
 // Export the Deployment name
 export const deploymentName = deployment.metadata.apply(m => m.name);
+
+// Create a LoadBalancer Service for the NGINX Deployment
+export const service = new k8s.core.v1.Service(name,
```

Comment:
I was doing this for users who cared to see the actual k8s spec, thinking someone new to pulumi from k8s would find it informative, instead of say just seeing a generated resource name. I could certainly remove it if it feels extraneous.

Reply:
I've gone ahead and removed the k8s resources from being exported.